### PR TITLE
Project news site

### PR DESCRIPTION
**Pull request by annathunberg**
**Millenial by Anna Thunberg**

**Struggled with:** flexbox, tried making the site using flexbox first, but started over with grid because I felt it was more logical.
I had some issues with the layout (width) of my small grid boxes as well, but I'm pretty happy with it anyways!

**Most poud of:** That I started early and kept going. That I was organized and commented out different parts of the code for readability (for myself). I'm proud of the process of making a clear visual idea into a real site, and the general appearance of my design.

Link to my project:
https://millenial.netlify.app/

### CODE REVIEW

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,342 @@
...
+  }
+
+  .socials {
+    display: inline-block;
+    margin-right: 30px;
+  }
+  
+}
+
+/* ----------------------- LARGE: DESKTOP SIZE -------------------------------- */
```
Comment (waliem) [inline code/style.css:220]: I'm not yet very 100% clear of the media queries myself, but I'm a little curious about what happens between the different queries. When I noticed that a specific width between 1100px and 1400 the smaller news items change the shape and becomes more of a vertical rectangle. was it to fit the width of the big news grid better in the view? :)

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,342 @@
...
+
+  .socials {
+    display: inline-block;
+    margin-right: 150px;
+  }
+  
+}
+
+
+/* ----------------------------- EXTRA LARGE: WIDE DESKTOP SIZE ---------------------------- */
```
Comment (waliem) [inline code/style.css:284]: apart from that you have a very well done media queries! I'm very impressed!!

**Review by waliem — COMMENTED** [review]
Hi Anna! Very very cool and nice looking magazine site you did! I'm so impressed that you managed to throw this together so fast and early on. I Loved the overall design, styling and effect you put into it. I even learned a a couple of things that I haven't tried myself before which is inspirational! you have such clear and easy to read structure of your code, which I really enjoyed to read through. You definitely meet the requirement of this project, you should be proud of yourself!  

ps. one thing that could be nice to read in the readme would be if you had any specific challanges or struggles big or small and how you solved them. But maybe you didn't have so much of that this time. Just a thought! :)

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,159 @@
...
+            <p>I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness... <a href="https://www.buzzfeed.com/anthonyrivas/whats-your-antidepressant-iq">[ read full story ]</a> </p>
+            </div>
+
+       <!-- SMALL NEWS  -->
+
+
+            <div class="grid-item"> 
+
+
+            <h2>Britneys dad is finally cancelled #freebritney</h2>
```
[the remark this answers is not in the document]
Comment (annathunberg) [reply code/index.html:91]: Yes, I added small pics but removed them again as I felt I didn't have time to style them in a way that looked nice.. Might bring them back 👀

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,342 @@
...
+  0% {
+      background-position: 0%;
+  }
+  100% {background-position: 100%;}
+}
+
+
+/* HEADER */
+
+.millenial-header {
```
[the remark this answers is not in the document]
Comment (annathunberg) [reply code/style.css:38]: YES! True, will definitely do this!

File: code/style.css (hunk 1 above)
Comment (annathunberg) [reply code/style.css:220]: Yes, I just thought it looked better layout-wise (honestly had to go back and inspect this myself 😅).

**Pull request closed without merging**